test/e2e/common/junit/junit.go

```diff
@@ -0,0 +1,11 @@
+package junit
+
+import (
+	"os"
+)
+
+func ConstructJunitFileName(name string) string {
+	reportDir := os.Getenv("e2e_reports_dir")
```

Comment:
Please check that `e2e_reports_dir` is not empty and called hasn't forgotten to set it. I'd say `panic()` is good enough here?

Reply:
Ok, check now added.